clients/java/org/dash/platform/dapi/v0/CoreGrpc.java

```diff
@@ -63,6 +63,18 @@ private CoreGrpc() {}
           .setResponseMarshaller(io.grpc.protobuf.ProtoUtils.marshaller(
               org.dash.platform.dapi.v0.CoreOuterClass.UpdateStateTransitionResponse.getDefaultInstance()))
           .build();
+  @io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/1901")
```

Comment:
Is that ok that it says "Experimental API"? :slightly_smiling_face:

Reply:
It's strange because it is an autogenerated part of code...